tests/unit/backend/utils.py

```diff
@@ -0,0 +1,21 @@
+from backend.corpora.common.corpora_orm import (
+    ProcessingState,
+    ValidationState,
+    ProjectStatus,
+)
+
+
+class ProjectParams:
```

Comment:
Is this purpose of this class to create a bogus Project entity?

Reply:
yes